pyclesperanto_prototype/_tier8/_AffineTransform3D.py

```diff
@@ -157,20 +157,22 @@ def shear_z(self,angle_x_in_degrees: float = 0, angle_y_in_degrees: float = 0 ):
         """          
         import math
         try:
-            shear_factor_xz = 1.0 / math.tan(angle_x_in_degrees * math.pi / 180)
+            shear_factor_xz = 1.0 / math.tan((angle_x_in_degrees-90) * math.pi / 180)
         except ZeroDivisionError:
-            shear_factor_xz = 0
+            print("Error1")
+            shear_factor_xz = math.pi
         
         try:
-            shear_factor_yz = 1.0 / math.tan(angle_y_in_degrees * math.pi / 180)
+            shear_factor_yz = 1.0 / math.tan((angle_y_in_degrees-90) * math.pi / 180)
```

Comment:
The way I've been calculating shear factor is by the equation:
`np.tan((90-angle) * np.pi / 180)`
Shouldn't you be using: 
`1.0 / math.tan((angle_x_in_degrees) * math.pi / 180)` ?? 
Whats the logic behind using `angle_y_in_degrees-90` ?

Reply:
The logic behind -90 degrees is that an angle of 0 should not shear. Or am I thinking wrong here?
You can see that in the small for-loops of the example notebook.

Reply:
90 - angle might do the same thing though...